spec/util/xcactivity_log_yaml_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'bundler/setup'
+require 'yaml'
+
+# Helper to sanitize the generated YAML due to different results on
+# CI vs local
+module XCActivityLogYAMLHelper
+  def sanitized_yaml(parsed_log)
```

Comment:
@segiddins @gyfelton `rake spec` was working locally and failing on CI. I noticed that the output of `YAML.dump(parsed)` in `xcactivitylog_spec.rb` was different for some reasons. 

Maybe it's a macOS vs Linux thing? I ran locally with the same versions of everything (`ruby`, `bundler`, etc) and the dumped yaml files were still different

I added this is fix the issues that I've found right before generating the yaml, so now regarding of what each call to `YAML.dump(parsed)` is doing natively the output is the same. All yaml files in `specs/fixtures/xcactivitylog` had to be re-generated to adopt the new pattern.

Let me know if this makes sense or if there's a better way/something I'm missing here

Reply:
:lolsob: